README.md

```diff
@@ -10,12 +26,6 @@ This is the Circulation Manager for [Library Simplified](https://www.librarysimp
 
 It depends on [Library Simplified Server Core](https://github.com/NYPL-Simplified/server_core) as a git submodule.
 
-## Installation
```

Comment:
This section was removed in favor of linking directly to the NYPL Simplified Wiki in the Appendix.